### PR TITLE
Compatibility with Rails 5.0

### PR DESCRIPTION
**Pull request by pschambacher**
:octopus::computer:

/cc @grosser 

### Description
Nothing special, worked straight away.

### Steps to merge
* [ ] :+1: from the team

### Risks
* None

### CODE REVIEW

File: gemfiles/rails3.2.gemfile [hunk 1]
```diff
@@ -3,3 +3,4 @@ source "https://rubygems.org"
 gemspec :path=>"../"
 
 gem "actionmailer", "~> 3.2.18"
+gem "test-unit"
```
Comment (grosser) [inline gemfiles/rails3.2.gemfile]: needed for ruby 2.2 ? -> test-unit-minitest ?

**Comment by grosser** [comment]
:+1:

File: gemfiles/rails3.2.gemfile (hunk 1 above)
Comment (pschambacher) [inline gemfiles/rails3.2.gemfile]: That the one that gets prompted now when trying to run rails 3.2 and ruby 2.2...

Comment (grosser) [inline gemfiles/rails3.2.gemfile]: yeah, adding  `test-unit-minitest` fixes it too and does not add a test-unit ... basically it's the minitest part that got removed from 2.2

**Comment by pschambacher** [comment]
I think your red build is coming from migration_tools. I reverted the change but I can't release. Would you mind releasing and adding zendesk and me on the gem?

**Pull request merged**